src/map/clif.cpp

```diff
@@ -14757,6 +14757,28 @@ void clif_parse_GM_Item_Monster(int fd, map_session_data *sd)
 	}
 }
 
+/// /resetcooltime 
+/// 0a88 (CZ_CMD_RESETCOOLTIME).
+void clif_parse_GMCooldownReset(int fd, map_session_data* sd) {
+#if PACKETVER >= 20160622
+	if(sd->group_id < 99)
```

Comment:
It would be better to create an atcommand and forward /resetcooltime to @resetcooltime so that the permission/group check can be customized via atcommands.yml

Reply:
i'will